operations/helm/charts/mimir-distributed/templates/_helpers.tpl

```diff
@@ -455,7 +455,7 @@ Examples:
   "tokengen" "tokengenJob"
 -}}
 {{- $componentSection := index $componentsMap .component -}}
-{{- if not $componentSection -}}{{- printf "No component section mapping for %s not found in values; submit a bug report if you are a user, edit mimir.componentSectionFromName if you are a contributor" .component | fail -}}{{- end -}}
+{{- if not $componentSection -}}{{- $componentSection := .component -}}{{- end -}}
```

Comment:
this might leave subtle bugs that we won't find out until someone complains. For example because some specialized value isn't set in our other tests and is missed during code review.

 It's clunky to edit this code, but it prevents bugs as it is